src/main.ts

```diff
@@ -144,6 +144,14 @@ export class Minifier {
 
         return this.contextEmit(node);
       }
+      case ts.SyntaxKind.PropertySignature: {
+        if (node.parent.kind === ts.SyntaxKind.TypeLiteral ||
+            node.parent.kind === ts.SyntaxKind.InterfaceDeclaration) {
+          let parentSymbol = this._typeChecker.getTypeAtLocation(node.parent).symbol;
+          return this.contextEmit(node, true);  // renameIdent is true for now
```

Comment:
replace comment with something that make more sense to people that might not have been part of our discussion. Something like `//TODO() incorporate check for assignment to an external type.`